ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -247,7 +247,7 @@ multiFieldRelevanceFunction
 
 /** tables */
 tableSource
-    : qualifiedName
+    : (catalogName DOT)? qualifiedName
```

Comment:
`qualifiedName` is `ident (DOT ident)*` which already takes care of the catalog name, right? I'm thinking it may be better to let `QualifiedName` class as the abstraction to hide the complexity, so no need to pass around one more catalog name.

Reply:
There will be additional syntax addition in next PR where catalogName is compulsory so brought this abstraction at syntax layer itself. 

```
tableSource
    : catalogName DOT nativeQueryFunction
    | (catalogName DOT)? qualifiedName
    | ID_DATE_SUFFIX
    ;

```

Reply:
I assume the `nativeQueryFunction` is similar as JDBC expression in Solr streaming expression: https://solr.apache.org/guide/6_6/stream-sources.html#jdbc-syntax. Is it more natural to move catalogName to an attribute in nativeQueryFunction? `catalog.function` looks a little weird.